src/components/Details/TranscriptFull/TranscriptFull.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC, RefObject } from "react";
+
+import TranscriptItem, { TranscriptItemRef } from "../TranscriptItem/TranscriptItem";
+
+import { Sentence } from "../../Shared/Types/Transcript";
+import hhmmss from "../../../utils/hhmmss";
+
+export interface TranscriptFullProps {
+  /**The sentences of the transcript */
+  sentences: Sentence[];
+  /**List of transcript item React references */
+  transcriptItemsRefs: RefObject<TranscriptItemRef>[];
+  /**Callback to play video clip */
+  jumpToVideoClip(startTime: number): void;
+}
+
+/**Full view of transcript */
+const TranscriptFull: FC<TranscriptFullProps> = ({
+  sentences,
+  transcriptItemsRefs,
+  jumpToVideoClip,
+}: TranscriptFullProps) => {
+  const onVideoClip = (startTime: number) => () => jumpToVideoClip(startTime);
```

Comment:
I don't particularly like this varaible name? Shouldn't this be "onVideoClick"? or maybe "setCurrentVideoPlaybackTime"?

Reply:
I'll change it to handleJumpToVideoClip and a comment that it's a function creator, create a function that handles jumping to video clip at startTime.